src/components/probeDetails.jsx

```diff
@@ -0,0 +1,380 @@
+    console.error('wrong dashType');
+    return '';
+  }
+
+  if (!['histogram', 'scalar', 'simpleMeasurements'].includes(type)) {
+    return '';
+  }
+
+  // The aggregator/TMO data uses different naming schemes for non-histograms probes.
+  if (type === 'scalar') {
+    name = 'SCALARS_' + name.toUpperCase();
+  } else if (type === 'simpleMeasurements') {
+    name = 'SIMPLE_MEASURES_' + name.toUpperCase();
+  }
+
+  return `https://telemetry.mozilla.org/new-pipeline/${dashType}.html#!` +
+          `max_channel_version=${channel}%252F${max_version}&`+
+          `min_channel_version=${channel}%252F${min_version}&` +
+          `measure=${name}` +
+          `&product=Firefox`;
+}
+
+
+function getProbeDocumentationURI(type) {
+  const sourceDocs = 'https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/';
+  const links = {
+    environment: sourceDocs + 'data/environment.html',
+    histogram: sourceDocs + 'collection/histograms.html',
+    scalar: sourceDocs + 'collection/scalars.html',
+    event: sourceDocs + 'collection/events.html',
+  };
+
+  return links[type] || sourceDocs;
+}
+
+// ported from explore.js (was getDatasetInfos())
+function getDatasetInfo(revisions, channelInfo, datasetMappings, probeId, probe, channel, state) {
+  const last = array => array[array.length - 1];
+
+  // Available documentation.
+  const dataDocs = {
+    longitudinal: 'https://docs.telemetry.mozilla.org/concepts/choosing_a_dataset.html#longitudinal',
+    main_summary: 'https://docs.telemetry.mozilla.org/concepts/choosing_a_dataset.html#mainsummary',
+    events: 'https://docs.telemetry.mozilla.org/datasets/batch_view/events/reference.html',
+  };
+
+  // Helper for code markup.
+  const code = s => <span className="code">{s}</span>;
+
+  const stmoLink = <a href="https://sql.telemetry.mozilla.org">STMO</a>;
+  const datasetInfo = [];
+
+  // TMO dashboard links.
+  if (['histogram', 'scalar'].includes(probe.type) ||
+      (probe.type === 'simpleMeasurements' && ['number', 'bool'].includes(state.details.kind))) {
+    const versions = getVersionRange(revisions, channelInfo, channel, state.revisions);
+    const distURL = getTelemetryDashboardURL('dist', probe.name, probe.type, channel, versions.first, versions.last);
+    datasetInfo.push(getNewTabLink(distURL, 'Measurements Dashboard'));
+  }
+
+  // Use counter dashboard links.
+  if ((probe.type === 'histogram') && probe.name.startsWith('USE_COUNTER2_')) {
+    const base = 'https://georgf.github.io/usecounters/';
+    const tokenizedName = probe.name.split('_');
+    const params = {
+      group: tokenizedName[2],
+      kind: last(tokenizedName).toLowerCase(),
+    };
+    const url = base + '#' + jqParam(params);
+    datasetInfo.push(getNewTabLink(url, 'Use Counter Dashboard'));
+  }
+
+  // Link to the hardware report for all hardware & OS related probes.
+  if (probeId.startsWith('environment/system.')) {
+    const url = 'https://hardware.metrics.mozilla.com/';
```

Comment:
That's the old URL and it may not be around for long. The new one is:

https://data.firefox.com/dashboard/hardware